### DISCUSSION TITLE
Add exponential ticker formatting option to log scaled axis

### DISCUSSION THREAD

**Discussion by sed-i** [grafana/grafana#49335] [Dashboards, Panels & Visualizations]
When using logarithmic scale
![Screenshot from 2022-05-20 11-06-01](https://user-images.githubusercontent.com/82407168/169557390-4c9a901a-b1ad-4920-9e40-fcda5666bb20.png)

it could be handy if there was a checkbox for formatting the ticks as exponents, so instead of

![Screenshot from 2022-05-20 10-30-00](https://user-images.githubusercontent.com/82407168/169557170-d0260cbb-e162-4cfd-8bd7-8dee11f07d11.png)

the Y axis would read
- 10^{1} MB/s
- 10^{0} MB/s
- 10^{-1} MB/s
- 10^{-2} MB/s
- 10^{-3} MB/s

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
